### ISSUE TITLE
Provide rendered view of tables in gitbook

### ISSUE THREAD

**Issue by jgvictores** [roboticslab-uc3m/teo-software-manual#32]
Provide rendered view of tables in gitbook

**Comment by jgvictores** [comment]
Blocked by: https://github.com/roboticslab-uc3m/questions-and-answers/issues/63

**Comment by jgvictores** [comment]
Not longer blocked by: https://github.com/roboticslab-uc3m/questions-and-answers/issues/63

**Comment by jgvictores** [comment]
Almost a duplicate of https://github.com/roboticslab-uc3m/developer-manual/issues/20

**Comment by jgvictores** [comment]
Most important tables migrated to `.csv`, and `gitbook-plugin-include-csv` preview added at https://github.com/roboticslab-uc3m/teo-developer-manual/commit/c8c3fc1ad3150a670ff0b9a05b819cd8acae0760
